natgeo/mortar
# [SYTO-1141] Updating playlist heading sizes at various breakpoints

https://jira.natgeo.com/browse/SYTO-1141

Updating playlist heading sizes at various breakpoints and altering heading margin/paddings at the mobile breakpoint

To test:
- Pull SYTO-1141
- `npm test`
- `gulp serve`
- navigate to http://localhost:3000/components/#video-playlist
- verify spacing changes and font size changes
